Assets/_GameJamCat/Scripts/PlayerController.cs

```diff
@@ -36,7 +57,8 @@ protected void Update()
 
             if (playerCharacter)
                 playerCharacter.Simulate(characterController, input);
-
+            if (_cameraAnimationInProgress == false)
```

Comment:
if (!_cameraAnimationInProgerss)

Reply:
Having == false is more convenient for readability imo